smartcosmos-framework-monitoring/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>net.smartcosmos</groupId>
+        <artifactId>smartcosmos-framework-parent</artifactId>
+        <version>3.0.1-SNAPSHOT</version>
+        <relativePath>../smartcosmos-framework-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>smartcosmos-framework-monitoring</artifactId>
```

Comment:
You're right. That was an open question: I wasn't sure about different versions for Framework modules.

If it get a new one, I would prefer 0.0.1-SNAPSHOT.